state/binarystorages.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 Canonical Ltd.
```

Comment:
Note that this is a renaming of previous state/tools.go, with some changes. Unfortunately git failed to understand this.